index.html

```diff
@@ -295,6 +295,7 @@ <h1 class="form-text">I'm always interested in hearing about new projects, so if
                 nestor.oro.alvarez@gmail.com
             </p>
         </footer>
+        <div id="overlay"></div>
```

Comment:
- [ ] Goog job adding the overlay as suggested by the previous reviewer, but kindly ensure you add padding around the overlay to match the figma design 👍🏽 

Figma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216165371-8596781a-3b49-4bb8-a55c-e5353c2095cf.png)

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/216165504-2ccd63c8-21ac-4316-ab56-8b458e067419.png)